.github/workflows/CI.yml

```
@@ -72,7 +72,7 @@ jobs:
         if: contains(matrix.os, 'windows')
         shell: cmd
         run: |
-            set CONDA_INSTALL_LOCN=C:\\Miniconda
+            set CONDA_INSTALL_LOCN=C:\\Miniconda3
```

I am not sure what the correct answer is here, but since the master works, let's revert to it:
```suggestion
            set CONDA_INSTALL_LOCN=C:\\Miniconda
```